README.md

```diff
@@ -44,7 +44,7 @@ Run notes
   and copied to /etc/udev/rules.d/
 * You should install NTP support on the system and configure time/date to automatically
   sinchronize to time servers.
-* Modify stget.sh for your space-track.org login and password (--post-data='identity=login&password=password')
+* Provide your <space-track.org> credentials for `stget.sh` via the environment variables `ST_USERNAME` and `ST_PASSWORD`
```

Comment:
In strf I use an $ST_LOGIN environment variable which holds both username and password. Let's use that here as well: https://github.com/cbassa/strf/blob/3acb78914db1d9e13307e96829f948a1e131b6a0/tleupdate#L15